src/Background.js

```diff
@@ -10,30 +10,35 @@ const styles = {
         position: "absolute",
         width: "100%",
         zIndex: props => props.ui.backgroundLayer,
-        "& $backgroundCell:nth-child(2n)": {
-            borderLeft: "dashed 1px #dfdfdf"
-        },
         height: "100%"
     },
     backgroundCell: {
         width: "100%",
         height: "100%",
-        borderLeft: "solid 1px #bcbcbc",
+        flexBasis: "1px",
+        boxSizing: "border-box"
     },
+    solidLeft: { borderLeft: "solid 1px #bcbcbc" },
+    dashedLeft: { borderLeft: "dashed 1px #dfdfdf" }
 }
 
 @inject("ui", "schedule")
 @injectSheet(styles)
 @observer
 class Background extends React.Component {
+    getBorderClass(cell, idx) {
+        if (idx === 0 || cell.time.get("minutes") === 0) return "solidLeft";

```

Comment:
I need the idx === 0 now because now it is possible to start with a half-hour block such as 8:30-9:00, and that would show up as a dashed left border. But we want the leftmost left border to always be solid (it separates shifts from the student name column).